src/api/system/system.ts

```diff
@@ -200,6 +201,13 @@ import { _Window } from '../window/window';
  * @property { string } uuid The uuid of the application
  */
 
+ /**
+ * InstalledRuntimes interface
+ * @typedef { object } InstalledRuntimes
+ * @property { string } action The name of action: "get-installed-runtimes"
```

Comment:
Is this being returned back to the end users?

Reply:
Yes, you can see the tutorial for an example. This API is meant to act in the same way as `getRVMInfo`